default.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+  <remote name="origin" fetch="https://git.launchpad.net/~snappy-hwe-team/snappy-hwe-snaps/+git" />
+  <default revision="master" remote="origin"  sync-j="4" />
+  <project path="en/stacks/audio/alsa" name="alsa-utils" />
+  <project path="en/stacks/bluetooth" name="bluez" />
+  <project path="en/stacks/modem" name="modem-manager" />
+  <project path="en/stacks/network" name="network-manager" />
```

Comment:
wifi-ap needs to go in here as well. In general I feel better about a combined networking chapter which would include network-manager, modem-manager, ofono, wifi-ap ..